WDL2CWL_example/samtoolsDocker/samtoolsDocker.cwl

```diff
@@ -0,0 +1,203 @@
+#!/usr/bin/env cwl-runner
```

Comment:
Can you share the purpose of this workflow?

Reply:
OK sir.